packages/terra-date-input/src/DateInputUtil.js

```diff
@@ -77,18 +77,17 @@ class DateInputUtil {
   static incrementYear(year) {
     // Handle empty input value
     if (year === '') {
-      return '1';
+      return DateInputUtil.MinYearValue.toString();
```

Comment:
If year is not an empty string or is not defined, we return null. Should it always return the min year value if undefined/null?

Reply:
On incrementing a null/undefined value it starts from 1 so changed it to start from 1900.

Reply:
Is this null return necessary then? https://github.com/cerner/terra-framework/pull/1261/files/9f1f78c7cefb5afde6ab25888703d2490c19a16d#diff-b5cce12f1b45fdee1f1ddd3837cbc55c5088424ddf957de7a28046fe64ecf1b4R98